### DISCUSSION TITLE
Is there a way to hide the caller ID?

### DISCUSSION THREAD

**Discussion by eugenefvdm** [mikopbx/Core#774] [General ( Обо всём )]
In our country South Africa, land lines start with 021....
Mobile phones start with 082....

We can only dial 021 at the moment. We cannot dial 082 calls.

We contacted the SIP provider who suggested to try turn off caller ID.

Is there a way with MikoPBX to turn off caller ID globally?

**Comment by boffart** [comment]
Try "Open your Provider" "Advanced setting" - "Redefining SIP header "From"" 
Tty set "user" field or "To disable the use of fromuser field"
